src/main/java/org/ambraproject/rhino/service/impl/SyndicationCrudServiceImpl.java

```diff
@@ -145,18 +145,25 @@ public Syndication createSyndication(ArticleRevisionIdentifier revisionId, Strin
     }
 
     return hibernateTemplate.execute(session -> {
-      Query query = session.createQuery("" +
-          "SELECT s " +
-          "FROM Syndication s " +
-          "JOIN s.articleVersion av " +
-          "JOIN av.journals j " +
-          "WHERE j.journalKey = :journalKey " +
-          "AND s.status in (:statuses)" +
-          "AND s.lastModified > :startTime");
-      query.setParameter("journalKey", journalKey);
-      query.setParameterList("statuses", statuses);
-      query.setDate("startTime", Date.from(startTime));
-      return (Collection<Syndication>) query.list();
+      Query query = session.createQuery(
+          "SELECT s FROM Syndication s " +
+              "JOIN s.articleRevision ar " +
+              "JOIN ar.ingestion ai " +
+              "JOIN ai.journal j " +
+              "WHERE j.journalKey = :journalKey " +
+              "AND s.status in (:statuses) " +
+              "AND s.lastModified > :startTime")
+          .setParameter("journalKey", journalKey)
+          .setParameterList("statuses", statuses)
+          .setDate("startTime", Date.from(startTime));
+
+      Collection<Syndication> result = (Collection<Syndication>) query.list();
+      // strikingImage causes gson serialization to go in a loop.
```

Comment:
If striking image serialization is an issue, it should be handled in `SyndicationOutputView.java`, rather than in the CRUD Service.